litmus-portal/frontend/src/pages/LoginPage/index.tsx

```diff
@@ -1,62 +1,122 @@
-import { Typography, Paper, TextField, Hidden } from '@material-ui/core';
+import {
+  Typography,
+  Paper,
+  TextField,
+  Hidden,
+  Button,
+} from '@material-ui/core';
 import { Link } from 'react-router-dom';
-import React from 'react';
-import ButtonFilled from '../../components/ButtonFilled/index';
+import React, { useState } from 'react';
 import useStyles from './styles';
 
+interface authData {
+  username: string;
+  password: string;
+}
+
 const LoginPage = () => {
   const classes = useStyles();
+  const [authData, setAuthData] = useState<authData>({
+    username: '',
+    password: '',
+  });
+  const handleForm = () => {
+    const formData: HTMLFormElement | null = document.querySelector(
+      '#login-form'
+    );
+    const data = new FormData(formData as HTMLFormElement);
+    const username = data.get('username') as string;
+    const password = data.get('password') as string;
+    data.append('username', username);
+    data.append('password', password);
+    fetch('/login', {
+      method: 'POST',
+      body: data,
+    }).then((response) => {
+      response.text();
+    });
+  };
   return (
     <div className={classes.rootContainer}>
       <div className={classes.root}>
         <div className={classes.rootDiv}>
           <div className={classes.mainDiv}>
             <img src="icons/LitmusLogo.png" alt="litmus logo" />
             <Typography variant="h2" className={classes.heading}>
-              Welcome to <strong>Litmus</strong>
+              Welcome to <strong>Litmus!</strong>
             </Typography>
             <Typography className={classes.description}>
               {' '}
               Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
               eiusmod tempor incididunt ut labore et dolore magna aliqua
             </Typography>
-            <div className={classes.inputDiv}>
-              <Paper className={classes.inputArea}>
-                <TextField
-                  id="filled-email-input"
-                  label="Email Address"
-                  InputProps={{ disableUnderline: true }}
-                  data-cy="inputEmail"
-                />
-              </Paper>
-              <Paper className={classes.inputArea}>
-                <TextField
-                  id="filled-password-input"
-                  label="Password"
-                  type="password"
-                  autoComplete="current-password"
-                  InputProps={{ disableUnderline: true }}
-                  data-cy="inputPassword"
-                />
-              </Paper>
+            <form
+              id="login-form"
+              className={classes.root}
+              autoComplete="on"
+              onSubmit={(event) => {
+                event.preventDefault();
+                handleForm();
+              }}
+            >
+              <div className={classes.inputDiv}>
+                <Paper className={classes.inputArea}>
```

Comment:
Is the Paper component necessary? Try [this](https://material-ui.com/components/text-fields/)

Reply:
[x] Updated

Reply:
* [x] Updated